server/apps/immich/src/main.ts

```diff
@@ -46,12 +46,12 @@ async function bootstrap() {
     customSiteTitle: 'Immich API Documentation',
   });
 
-  // Generate API Documentation
```

Comment:
I was planning to move this to development only but forgot to do that. Thank you